backend/api/response.go

```diff
@@ -0,0 +1,36 @@
+package api
+
+import (
+	"time"
+
+	"github.com/gin-gonic/gin"
+	db "github.com/git-adithyanair/cs130-group-project/db/sqlc"
+)
+
+// ========================================================================
+// Function to convert errors into a form that can be sent as a response.
+func errorResponse(err error) gin.H {
+	return gin.H{"error": err.Error()}
+}
+
+// ========================================================================
+
+// ========================================================================
+// Type and function to return only unprotected user information.
+type userResponse struct {
+	ID        int64     `json:"id"`
+	Email     string    `json:"email"`
+	FullName  string    `json:"full_name"`
+	CreatedAt time.Time `json:"created_at"`
```

Comment:
Do we not have to return the token in the response here? Or at what point does the token get sent in the auth process?